runtime/runtime-domain-service/docker/gradle.properties

```diff
@@ -0,0 +1 @@
+docker_image_name = runtime-api
```

Comment:
Shall we remove this file and move this property to gradle.properties file in the root project. It's a best practice as in [gradle docs](https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#declare_properties_in_gradle_properties_file)?

> A typical project setup places the gradle.properties file in the root directory of the build.